src/common/core/cli/healthcheck.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import socket
+
+
+def get_args(argv: list[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
```

Comment:
Maybe this is overkill at this point, but I wonder if we should explicitly say what type of health check this is in a subcommand or flag.

For example, we could do something like:

```bash
flagsmith healthcheck tcp
```

```bash
flagsmith healthcheck http --path /health/liveness
```

This would let us have different sets of options for different types of health checks, instead of having them all together in one command.